static-site/data/BaseConfig.js

```diff
@@ -30,3 +30,13 @@ export const blogFeeds = {
   rss2: `rss2.xml`,
 }
 export const blogFeedUrls = Object.fromEntries( Object.entries(blogFeeds).map(([k,v]) => [k, `${blogUrl}/${v}`]));
+
+/*
+  See <https://plausible.io/docs/plausible-script>. Analytics are
+  disabled on localhost, but use `script.local.js` if you want to
+  track localhost for testing purposes. If you want to disable all
+  analytics, set both of these constants to `undefined`.
+*/
+export const AnalyticsSource = "https://plausible.io/js/script.js";
+export const AnalyticsDomain = "nextstrain.org";
```

Comment:
I'd suggest

```suggestion
export const analyticsSource = "https://plausible.io/js/script.js";
export const analyticsDomain = "nextstrain.org";
```

to match the rest of the vars in this file.

And maybe also make the names specific to Plausible since they're tightly coupled to it?


```suggestion
export const plausibleAnalyticsScript = "https://plausible.io/js/script.js";
export const plausibleAnalyticsDomain = "nextstrain.org";
```

Reply:
Fixed in force-push.